### ISSUE TITLE
Trouble with URL parsing

### ISSUE THREAD

**Issue by kgizdov** [apas/pastery#5]
Hi,

I'm having the following error when I try to use pastery:

``` python
Trying to post with Python lib
Trying to post with CURL
Traceback (most recent call last):
  File "pastery in /home/gizdov/.config/sublime-text-3/Installed Packages/Pastery.sublime-package", line 23, in run
TypeError: string argument without an encoding

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/sublime_text_3/sublime_plugin.py", line 556, in run_
    return self.run(edit)
  File "pastery in /home/gizdov/.config/sublime-text-3/Installed Packages/Pastery.sublime-package", line 40, in run
KeyError: 'url'
```

By setting the encoding in line 23 the issue is resolved, but another is revealed.

``` python
req = Request(url,
            data=bytes(content, 'utf8'),  # specifying encoding as per python docs
            headers={'User-Agent': 'Mozilla/5.0 (Sublime Text) Pastery plugin'})
```

This resolves the error, however we get a new error:

``` python
Trying to post with Python lib
Trying to post with CURL
Traceback (most recent call last):
  File "/home/gizdov/.config/sublime-text-3/Packages/Pastery/pastery.py", line 25, in run
    response = urlopen(req)
  File "./urllib/request.py", line 156, in urlopen
  File "./urllib/request.py", line 475, in open
  File "./urllib/request.py", line 587, in http_response
  File "./urllib/request.py", line 513, in error
  File "./urllib/request.py", line 447, in _call_chain
  File "./urllib/request.py", line 595, in http_error_default
urllib.error.HTTPError: HTTP Error 422: Unknown Status Code

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/sublime_text_3/sublime_plugin.py", line 556, in run_
    return self.run(edit)
  File "/home/gizdov/.config/sublime-text-3/Packages/Pastery/pastery.py", line 40, in run
    rurl = json.loads(stdout.decode("utf8"))["url"]
KeyError: 'url'
```

Turns out the dictionary that gets returned does not have a "url" key. Printing the keys shows this:

``` python
rurl = json.loads(stdout.decode("utf8"))["url"]
      print(rurl.keys())
>>> dict_keys(['result', 'error_msg'])
```

Comments?

**Comment by apas** [comment]
Fixed and merged in #6.

**Comment by kgizdov** [comment]
Thanks. Are you releasing it to PackageControl or should I just keep the manual install for now?
